ViewGenerator/build/ViewGenerator.targets

```diff
@@ -21,6 +21,10 @@
     
   </PropertyGroup>
 
+  <Target Name="GenerateBundle">
+    <Exec Command="&quot;$(ViewGeneratorNodeJsExe)&quot; &quot;$(ProjectDir)node_modules\webpack\bin\webpack.js&quot; --mode production" />
```

Comment:
webpack should not be in the project dir

Reply:
I will move this target to ViewGeneratorCore.targets, since webpack modules are now there